cli/src/main/java/org/jboss/as/cli/impl/aesh/cmd/security/model/ElytronUtil.java

```diff
@@ -0,0 +1,451 @@
+/*
+Copyright 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+package org.jboss.as.cli.impl.aesh.cmd.security.model;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.jboss.as.cli.CommandContext;
+import org.jboss.as.cli.Util;
+import org.jboss.as.cli.operation.OperationFormatException;
+import org.jboss.as.cli.operation.impl.DefaultOperationRequestBuilder;
+import org.jboss.as.controller.client.ModelControllerClient;
+import org.jboss.dmr.ModelNode;
+
+/**
+ * Utility class to interact with Elytron subsystem.
+ *
+ * @author jdenise@redhat.com
+ */
+public abstract class ElytronUtil {
+
+    public static final String JKS = "JKS";
+    public static final String PKCS12 = "PKCS12";
+    public static final String TLS_V1_2 = "TLSv1.2";
```

Comment:
Should we also support TLSv1 and 1.1?

Reply:
For now we are not exposing an option for it. This could be added if this is seen as a strong limitation.